lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -29,11 +29,11 @@
     </header>
 
     <ul class="wrapper-tabs">
-        <li class="tab tab-manage_students is-selected" data-tab="manage_students"><a href="#"><%- gettext("Manage Students") %></a></li>
+        <li class="tab tab-manage_students is-selected" data-tab="manage_students"><a href="#"><span class="sr"><%- gettext('Selected tab') %> </span><%- gettext("Manage Students") %></a></li>
```

Comment:
@andy-armstrong I'm unfamiliar with this issue. This is the first time anyone's brought up a potential RTL issue with i18n markup, at least in my knowledge.